src/api/reflector.rs

```diff
@@ -223,3 +224,32 @@ impl<K> Reflector<K> where
         Ok(())
     }
 }
+
+/// ObjectId represents an object by name and namespace (if any)
+#[derive(Ord, PartialOrd, Hash, Eq, PartialEq, Clone)]
+struct ObjectId {
+    name: String,
+    namespace: Option<String>,
+}
+
+impl ToString for ObjectId {
```

Comment:
I added this only for the debug message on: https://github.com/clux/kube-rs/pull/56/files#diff-01dd4282bc012d0f05763812f0312a9dR175

Reply:
That makes complete sense :+1: